simulation/src/world.rs

```diff
@@ -27,6 +28,7 @@ new_key_type! {
     pub struct WagonID;
     pub struct FreightStationID;
     pub struct CompanyID;
+    pub struct BirdID;
```

Comment:
Sorry for all the mess in this file, creating a new entity shouldn't be that much boilerplate. It can all be put under a macro at some point.